composer.json

```diff
@@ -29,7 +29,7 @@
     },
     "require-dev": {
         "phpstan/phpstan": "^1.11",
-        "phpunit/phpunit": "^9.6",
+        "phpunit/phpunit": "^10.5.29",
```

Comment:
There is no need for the patch version. `^10.5` will be enough

Reply:
Fixed